x/globalfee/ante/fee.go

```diff
@@ -158,3 +169,37 @@ func (mfd FeeDecorator) getBondDenom(ctx sdk.Context) string {
 
 	return bondDenom
 }
+
+// ContainsOnlyBypassMinFeeMsgs returns true if all the given msgs type are listed
+// in the BypassMinFeeMsgTypes of the FeeDecorator.
+func (mfd FeeDecorator) ContainsOnlyBypassMinFeeMsgs(msgs []sdk.Msg) bool {
+	for _, msg := range msgs {
```

Comment:
why this is moved from fee_utils.go to fee.go

Reply:
It just makes its unit testing easier since it reads data from the fee decorator.

Reply:
It just makes its unit testing easier since it reads data from the fee decorator.